sopel/plugins/__init__.py

```diff
@@ -62,12 +62,12 @@ def find_internal_plugins():
     """List internal plugins.
 
     :return: yield instances of :class:`~.handlers.PyModulePlugin`
-             configured for ``sopel.modules.*``
+             configured for ``sopel.builtins.*``
 
-    Internal plugins can be found under ``sopel.modules``. This list does not
+    Internal plugins can be found under ``sopel.builtins``. This list does not
     include the ``coretasks`` plugin.
     """
-    modules = importlib.util.find_spec('sopel.modules')
+    modules = importlib.util.find_spec('sopel.builtins')
```

Comment:
Nit: Seems to me that we could drop the name `modules` here as well. I would propose `builtins_spec` as a replacement name.